lib/hubspot/association.rb

```diff
@@ -0,0 +1,80 @@
+class Hubspot::Association
+  COMPANY_TO_CONTACT = 2
+  DEAL_TO_CONTACT = 3
+  CONTACT_TO_DEAL = 4
+  DEAL_TO_COMPANY = 5
+  COMPANY_TO_DEAL = 6
```

Comment:
I added the bare minimum, [the long list is long and is here](https://developers.hubspot.com/docs/methods/crm-associations/crm-associations-overview). 
I guess we can make this list grow with time?